tools/myst/common.c

```diff
@@ -186,3 +190,44 @@ bool myst_validate_mount_config(myst_mounts_config_t* mounts)
 
     return true;
 }
+
+int myst_generate_config_id(
+    const uint8_t* root_hash,
+    size_t root_hash_size,
+    const char* config_buffer,
+    size_t config_size,
+    uint8_t* config_id)
+{
+    int ret = 0;
+    myst_sha256_t sha256;
+    char* concat_buf = NULL;
+    // Omit the '\0' at the end of config buffer
```

Comment:
It's unclear how this contract is enforced. If we want to enforce this, I'd suggest adding a check like
```c
if (config_buffer[config_size - 1] != '\0')
  ERAISE(...);
```

Also, need to ensure `config_buffer` is not NULL and the `config_size` greater than 0.

Reply:
@mingweishih Thanks for reviewing. This is a temporary solution to append roothash to JSON buffer. In the final version, roothash will be inserted into the config JSON structure.

Reply:
In the product implementation, the roothash info will be in json name/value pair, as as the app's config.json entries, for example:

"rootfs-measurement-alg": "sha256-flat" 
"rootfs-measurement": “0x1F302E54382B30DFCC21A34B6678F320347890A92D3D45098720C5CCAA303123”